sherpa/fit.py

```diff
@@ -23,6 +23,9 @@
 import logging
 import os
 import signal
+
+from functools import wraps
```

Comment:
I was glad to find it available for both Python 2.7 and 3.5, as I didn't fancy writing the logic to copy over the function syntax as part of the decorator. We should probably review elsewhere we use decorators to see if we can use `functools.wraps`, but I decided that was not for this PR.